### DISCUSSION TITLE
High CPU usage when running iOS games

### DISCUSSION THREAD

**Discussion by Williamclouf** [PlayCover/PlayCover#865] [General]
The CPU usage is around 150% when running iOS/iPadOS games. So does some lightweight games such as Subway Surf and Hearthstone, causing the laptop overheating while some games available in Mac AppStore are really efficient.
Is this beacause the wrapper that imitates a mobile device that lots of computing resources are put into it?
And is there some way to optimize CPU usage in manipulation or foundation?

**Comment by ohaiibuzzle** [comment]
150% is actually not that high
It seems high but remember that 100% here is per-core, so on an 8 core M1 system, maximum system load is actually 800%

Someone did some testing (on Discord) and found PlayCover to have basically no impact on performance compared to a native macOS app in 3DMark
